.dockerignore

```diff
@@ -0,0 +1,9 @@
+/.idea
+/*.iml
+/.vscode
+
+/.dev
+/bin
+/dist
+/site
+/coverage.txt
```

Comment:
It seems that there are no such files in this repo?

Reply:
I've updated them fyi @ktock